docs/src/content/docs/ops/backup-and-restore/empty-log.adoc

```diff
@@ -0,0 +1,68 @@
+---
+title: Scenario - Empty Log & Intact Storage
+---
+
+If the log is unexpectedly empty — for example, due to topic deletion or reinitialization — but previously indexed storage still exists, XTDB will refuse to start. This is a safety mechanism designed to prevent state divergence between the log and the object store.
+
+You may encounter an error such as:
+
+```
+Starting node with an empty Kafka log topic and previously indexed values (latest completed offset: 289).
+```
+
+== What Is Preserved
+
+If the storage backend remains intact, then all previously compacted and indexed data is still available.
+
+* Historical data in all compacted Arrow files (Level-0 and higher)
+* The full queryable state of the database, up to the last successfully indexed offset
+
+== What Is Lost
+
+* The transaction log and its message history, including any unindexed or recently submitted transactions
```

Comment:
How something slightly more meaningful to the user: "Any recently submitted transactions that were not indexed prior to the loss of the transaction log"

Reply:
:+1: Seems good, thank you for the suggestion!